build-logic/basics/src/main/kotlin/gradlebuild/basics/kotlindsl/kotlin-dsl-upstream-candidates.kt

```diff
@@ -28,4 +28,7 @@ fun Project.execAndGetStdout(workingDir: File, vararg args: String): String {
 }
 
 
-fun Project.execAndGetStdout(vararg args: String) = execAndGetStdout(File("."), *args)
+fun Project.execAndGetStdout(vararg args: String) = execAndGetStdout(File("."), true, *args)
+
+
+fun Project.execSuccessfulAndGetStdout(vararg args: String) = execAndGetStdout(File("."), false, *args)
```

Comment:
```suggestion
fun Project.execIAndGetStdoutIgnoringError(vararg args: String) = execAndGetStdout(File("."), false, *args)
```

`ExecSuccessful` sounds confusing for me: "exec [and ensure] successful" vs "exec successful[ly in any case]."